Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM alpine:3
+
+# Variables set with ARG can be overridden at image build time with
+# "--build-arg var=value".  They are not available in the running container.
+ARG restic_ver=0.15.2
```

Comment:
Probably not significant, but it caught my eye that lines 5 and 6 are different in that one uses quotes and the other doesn't